specs/vscode_extensions.table

```diff
@@ -11,6 +11,7 @@ schema([
     Column("prerelease", INTEGER, "Pre release version"),
     Column("uid", BIGINT, "The local user that owns the plugin",
       additional=True),
+    Column("version_type", TEXT, "VSCode or VSCode Insiders")
```

Comment:
For `server`, we’re already exposing it for the regular VSCode version. If I’m running osquery on WSL and have VSCode on my Windows, the extensions come from the `/home/.../.vscode-server/extensions` path, for example.

And I agree `version_type` may not be the best name, as it could get confused with the extension’s version type. `vscode_edition` seems better. We could omit it, but I think having it as a single column adds clarity.

So maybe keep exposing `server` and change the column to `vscode_edition`?

Reply:
Yeah, I think keeping it, but naming it `vscode_edition` makes more sense.

Reply:
Sounds good, I kept it and changed the name to `vscode_edition`